pkg/gitproviders/provider.go

```diff
@@ -25,6 +25,8 @@ const (
 	defaultTimeout = time.Second * 30
 )
 
+var RepositoryNoPermissionsOrDoesNotExistError = errors.New("no permissions to access this repository or repository doesn't exists")
```

Comment:
major: errors are prefixed with the `Err` like in `gitprovider.ErrNotFound` and the error types are suffixed with the `Error`